src/rngs/os.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Developers of the Rand project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Interface to the random number generator of the operating system.
+
+use getrandom::getrandom;
+use rand_core::{CryptoRng, RngCore, Error, ErrorKind, impls};
+
+/// A random number generator that retrieves randomness from from the
+/// operating system.
+///
+/// This is a zero-sized struct. It can be freely constructed with `OsRng`.
```

Comment:
I think it's worth to add a usage example, as it may be unintuitive for some less experienced users that you can write `OsRng.next_u32()`.

Reply:
There are a couple of examples, but I'll take another look...

The CI still fails on cloudabi — probably we need to make a new release, but I guess it would be better to test the patch first.